### ISSUE TITLE
Using the model that trained on my own dataset as pretrained is sense?

### ISSUE THREAD

**Issue by Cestovatels** [ultralytics/ultralytics#15576]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Hello,

I have a car dataset that is constantly updated. Previously, I trained with 500 images and got a model. Now I have 1200 images in the dataset.  Does it make sense to use the old model as pretrained in the new training to increase model success? The new dataset has 500 images from the old dataset, will this have a negative impact?

Thank you for your answers.

**Comment by Y-T-G** [comment]
If the training data from previous training has entered into validation set (random splitting), then you shouldn't use it. If not, you can use it.
